source/wp-content/themes/wporg-showcase-2022/src/style/style.scss

```diff
@@ -127,3 +127,21 @@ button[type="submit"] {
 	}
 
 }
+
+@media (max-width: 600px) {
+	.wporg-results-bar-container {
+		flex-direction: column;
+	}
+
+	.wporg-results-bar-container > * {
+		width: 100%;
+	}
+
+	.wporg-results-bar-container > *:first-child {
+		order: 2;
+	}
+
+	.wporg-results-bar-container > *:last-child {
+		order: 1;
+	}
```

Comment:
Changing order like this breaks the expected pattern for keyboard navigation, but any other ideas to fix it and keep the visual display are pretty hacky… Not a blocker, but not great 🤷🏻

Reply:
Can you be more specific on how? Aren't the form controls the only focusable elements? Noting, the order will only be changed on mobile.

Reply:
Fair question, I wasn't clear - I said keyboard nav but was really talking about how the screen reader focus moves. A screen reader will read in source-order, so in this case the context bar will be read first, then it'll skip back to the search, then skip the context to get to the content. And that's the case for mobile screen readers too.

Reply:
It does! Well done.

I'm going to merge this now.